packages/fluentui/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 38 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27114&commitId=c5439df15858096d21e252c28680a8bcd7634983&env=prod)

✅ There was 18 screenshots added, 0 screenshots removed, 1222 screenshots unchanged, 0 screenshots with different dimensions and 20 screenshots with visible difference.


  <details><summary><b>unknown</b> 38 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Datepicker.Datepicker Cell Example Teams Dark.Does not show tooltip on not today's date..chromium.png|424| Changed|
|Datepicker.Datepicker Cell Example Teams Dark.Shows datepicker popup..chromium.png|424| Changed|
|Datepicker.Datepicker Cell Example Teams Dark.Shows tooltip on today's date..chromium.png|424| Changed|
|Datepicker.Datepicker Cell Example Teams High Contrast.Does not show tooltip on not today's date..chromium.png|505| Changed|
|Datepicker.Datepicker Cell Example Teams High Contrast.Shows datepicker popup..chromium.png|505| Changed|
|Datepicker.Datepicker Cell Example Teams High Contrast.Shows tooltip on today's date..chromium.png|505| Changed|
|Datepicker.Datepicker Cell Example Teams.Does not show tooltip on not today's date..chromium.png|435| Changed|
|Datepicker.Datepicker Cell Example Teams.Shows datepicker popup..chromium.png|435| Changed|
|Datepicker.Datepicker Cell Example Teams.Shows tooltip on today's date..chromium.png|435| Changed|
|Datepicker.Datepicker Example Clearable.Shows cleared calendar..chromium.png|387| Changed|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.Disabled date is not clickable..chromium.png|81| Changed|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.Month icon is not clickable..chromium.png|81| Changed|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.Shows min max dates..chromium.png|81| Changed|
|Datepicker.Datepicker Example Restricted Dates Teams High Contrast.Shows disabled dates..chromium.png|25| Changed|
|Dropdown.Dropdown Example Loading Teams Dark.chromium.png|642| Changed|
|Dropdown.Dropdown Example Loading Teams High Contrast.chromium.png|222| Changed|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Clicks on the last item and opens submenu.chromium.png|1138| Changed|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Focuses on the first element in the submenu.chromium.png|1196| Changed|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Clicks on the last item and opens submenu.chromium.png|1550| Changed|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Focuses on the first element in the submenu.chromium.png|1553| Changed|
|Menu.Menu Example Underlined Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams Dark.chromium.png|0| Added|
|Menu.Menu Example Underlined Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams High Contrast.chromium.png|0| Added|
|Menu.Menu Example Vertical Teams.Clicks on 4th item (opens submenu).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams.Hovers 1st item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams.Hovers 4th item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams.chromium.png|0| Added|
|Menu.Menu Example Vertical.Clicks on 4th item (opens submenu).chromium.png|0| Added|
|Menu.Menu Example Vertical.Hovers 1st item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical.Hovers 4th item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical.chromium.png|0| Added|

</details>


<div id="vrtCommentFluentUI-v0"/>